README.md

```diff
@@ -100,14 +100,14 @@ HUGO_UGLYURLS=false hugo serve -w
 > **Note:**  
 > "Ugly URLs" refer to URLs ending with `.html` (e.g., `/about.html`). GitHub Pages prefers these for compatibility. The `serve-public.js` script emulates this behavior.
 
-Pro-Tip: Do this in a sparse checkout that excludes large parts of `content/`, to speed up the rendering time.
+### Enabling Search Locally
 
-To test the site locally _with_ the search enabled, run this instead:
+To test the site with search enabled:
```

Comment:
Without stressing that this is happening locally, would readers not be confused into thinking that they test the site in place, i.e. at https://git-scm.com/?